packages/asc-assets/static/fonts/beta-fonts.css

```diff
@@ -0,0 +1,57 @@
+@import url("https://fast.fonts.net/t/1.css?apiType=css&projectid=88ed95f9-d020-4dee-be83-444c3ada1a0e");
```

Comment:
Shouldn't this file (`beta-fonts.css`) also be placed in a beta folder? Like so: `packages/asc-assets/beta/static/fonts/beta-fonts.css`?

Reply:
I dunno, does it have negative consequences to keep it where it is? We probably don't need anything else in the `beta/static` folder, so it would be a folder containing a single folder... Or maybe something like `packages/asc-assets/static/fonts/beta`?